Clinical-Genomics/scout
# Causatives complete info

fix #1169.

Added ACMG classification and filter by gene. Causatives page now looks like this: 

![image](https://user-images.githubusercontent.com/28093618/59584643-da3bc380-90de-11e9-8598-df99dcde3413.png)

Note how first variant has ACMG classification and second one doesn't.

How to test:
1. Assign any ACMG classification to a variant and mark it as causative.
1. Variant should be displayed among the causatives http://scout-url/custXXX/causatives (local implementation of scout: http://localhost:5000/cust000/causatives)
1. Make sure that the ACMG classification is displayed and the tooltip is displayed when the mouse hovers over it: 
![image](https://user-images.githubusercontent.com/28093618/59585335-89c56580-90e0-11e9-8d79-f27496f30e2b.png)
1. Mark another variant as causative.
1. Go to causatives (same page as above) and check that 2 variants are displayed.
1. Filter by gene using the gene of one of the 2 variants (variants should be in different genes)
1. Just one variant should now be displayed (the one falling in the queried gene)

- [x] Review by @dnil 
- [x] Verified by @dnil
